### PR TITLE
NVMApp Api

### PR DESCRIPTION
**Pull request by aaitor**
## Description

New NvmApp API oriented to a simpler integration of applications built around the NVM App

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

#### Funny gif

Put a link of a funny gif inside the parenthesis-->

### CODE REVIEW

File: src/keeper/contracts/templates/AgreementTemplate.abstract.ts [hunk 1]
```diff
@@ -286,6 +286,7 @@ export abstract class AgreementTemplate<Params> extends ContractBase {
       timeouts.push(condition.timeout)
       timelocks.push(condition.timelock)
     })
+    console.log(`-- Ordering with price: ${assetPrice.toString()}`)
```
Comment (r-marques) [inline src/keeper/contracts/templates/AgreementTemplate.abstract.ts:289]: cleanup

File: src/models/AssetPrice.ts [hunk 2]
```diff
@@ -22,10 +22,12 @@ export class AssetPrice {
     } else if (_params.length === 2) {
       const [address, amount] = _params
       this.rewards.set(address, amount)
+      // this.rewards[address] = amount
```
Comment (r-marques) [inline src/models/AssetPrice.ts:25]: cleanup

File: src/models/AssetPrice.ts [hunk 3]
```diff
@@ -22,10 +22,12 @@ export class AssetPrice {
...
       this.totalPrice = amount
     } else if (_params.length === 3) {
       const [address, amount, tokenAddress] = _params
       this.rewards.set(address, amount)
+      // this.rewards[address] = amount
```
Comment (r-marques) [inline src/models/AssetPrice.ts:30]: cleanup

File: src/nevermined/resources/AppNetworks.ts [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  instanceName: string
+  nftContractAddress?: string
+  tokenAddress?: string // ERC-20 token address or zero address (0x000..) for native token
+}
+
+export class AppDeploymentLocal extends NeverminedAppOptions {
+  instanceName = 'localnet'
+  web3ProviderUri = 'http://contracts.nevermined.localnet'
+  marketplaceUri = 'http://marketplace.nevermined.localnet'
+  graphHttpUri = 'http://localhost:9000/subgraphs/name/nevermined-io/development'
```
Comment (r-marques) [inline src/nevermined/resources/AppNetworks.ts:13]: maybe this should be undefined since we are not testing locally with subgraphs

**Review by eruizgar91 — APPROVED** [review]

**Pull request merged**